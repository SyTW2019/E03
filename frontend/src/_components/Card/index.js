import React from 'react'
import { makeStyles } from '@material-ui/core/styles'
import Card from '@material-ui/core/Card'
import CardActionArea from '@material-ui/core/CardActionArea'
import CardActions from '@material-ui/core/CardActions'
import CardContent from '@material-ui/core/CardContent'
import CardMedia from '@material-ui/core/CardMedia'
import Button from '@material-ui/core/Button'
import Typography from '@material-ui/core/Typography'

const useStyles = makeStyles({
  card: {
    maxWidth: 345,
    alignItems: 'center',
    margin: '0 20px',
  },
})
export default function ImgMediaCard(props) {
  const classes = useStyles()
  return (
    <Card className={classes.card}>
      <CardActionArea>
        <CardMedia
          component="img"
          alt="Libros"
          height="140"
          image={props.src}
          title="Libros"
        />
        <CardContent>
          <Typography gutterBottom variant="h5" component="h2">
            Event
          </Typography>
          <Typography variant="body2" color="textSecondary" component="p">
            Esto es una tarjeta para un evento. Aqui se inserta la descripcion y
            detalles del Evento.
          </Typography>
        </CardContent>
      </CardActionArea>
      <CardActions>
        <Button size="small" color="primary">
          Share
        </Button>
        <Button size="small" color="primary">
          Learn More
        </Button>
      </CardActions>
    </Card>
  )
}
