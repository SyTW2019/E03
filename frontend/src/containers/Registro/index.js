import React, { Component } from 'react';
import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
import axios from 'axios';
import SignUp from 'components/SignUp';

export default function Registro() {

  function nueva(user){
    axios.post('http://localhost:3000/usuario', user)
    .then(function (response) {
      console.log(response)
    })
    .catch(function (error) {
      console.log(error)
    });
   }

  return (
    <div>
      <SignUp action={nueva} />
    </div>
  );

}
