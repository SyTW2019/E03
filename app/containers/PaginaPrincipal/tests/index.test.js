import React from 'react';
import ReactDOM from 'react-dom';
import { render, cleanup } from 'react-testing-library';
import "jest-dom/extend-expect";
import renderer from "react-test-renderer";

import Card from '../../../components/Card/index'
import Registro from '../index'

describe('<Registro />', () => {

  afterEach(cleanup);

  it('render', () => {
      const div = document.createElement(div);
      ReactDOM.render(<Registro/>,div);
  });

  it('should render an Card component', () => {
    const { container } = render(<Card />);
    expect(container).not.toBeNull();
  });

  it('Matches snapshots', () => {
    const tree = renderer.create(<Registro title="save"/>).toJSON();
    expect(tree).toMatchSnapshot();
  });


});