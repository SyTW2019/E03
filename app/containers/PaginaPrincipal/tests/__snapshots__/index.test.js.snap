// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PaginaPrincipal /> Matches snapshots 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthXs"
>
  <div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
        >
          <button
            className="MuiButtonBase-root MuiCardActionArea-root"
            disabled={false}
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <img
              alt="Contemplative Reptile"
              className="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img"
              height="140"
              src="IMAGE_MOCK"
              title="Contemplative Reptile"
            />
            <div
              className="MuiCardContent-root"
            >
              <h2
                className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
              >
                Event
              </h2>
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Esto es una tarjeta para un evento. Aqui se inserta la descripcion y detalles del Evento.
              </p>
            </div>
            <span
              className="MuiCardActionArea-focusHighlight"
            />
          </button>
          <div
            className="MuiCardActions-root MuiCardActions-spacing"
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-textSizeSmall MuiButton-sizeSmall"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                Share
              </span>
            </button>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-textSizeSmall MuiButton-sizeSmall"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                Learn More
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
        >
          <button
            className="MuiButtonBase-root MuiCardActionArea-root"
            disabled={false}
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <img
              alt="Contemplative Reptile"
              className="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img"
              height="140"
              src="IMAGE_MOCK"
              title="Contemplative Reptile"
            />
            <div
              className="MuiCardContent-root"
            >
              <h2
                className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
              >
                Event
              </h2>
              <p
                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
              >
                Esto es una tarjeta para un evento. Aqui se inserta la descripcion y detalles del Evento.
              </p>
            </div>
            <span
              className="MuiCardActionArea-focusHighlight"
            />
          </button>
          <div
            className="MuiCardActions-root MuiCardActions-spacing"
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-textSizeSmall MuiButton-sizeSmall"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                Share
              </span>
            </button>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-textSizeSmall MuiButton-sizeSmall"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                Learn More
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</main>
`;
